lib/engine/game/g_18_usa/step/route.rb

```diff
@@ -68,6 +68,7 @@ def train_choices(entity, allow_permanents)
             @game.route_trains(entity).reject do |t|
               (!allow_permanents && !t.obsolete_on && !t.rusts_on) || t.variant['name'] == 'P'
```

Comment:
I know this line wasn't changed, but can you use @game.pullman_train?(t) instead of == 'P'?

Reply:
What's the purpose of these lines if the result is being ignored on line 71?

Reply:
Line 71 was added when I was debugging the issue but I forgot to remove that line. I changed 69 to use @game.pullman_train?(t)